utils/reader.py

```diff
@@ -1,9 +1,16 @@
-def read_file():
-    file_values = []
-    with open("/home/zia/ISDP/Hammad sab/Python/wether_Mar/weather task 2 in csv file/files/f1.csv") as file_read:
+def read_file(path_1, path_2):
+    file_values1 = []
```

Comment:
have your function which takes on paramater of file_path and return the files contents from that function. should be used like this
```python
file1_content = read_file(".../f1.csv")
file2_content = read_file(".../f2.csv")